packages/terra-form-select/src/tag/Menu.jsx

```diff
@@ -340,13 +340,27 @@ class Menu extends React.Component {
     }
 
     const {
-      input, onDeselect, onSelect, value,
+      input, onDeselect, onSelect, value, intl, visuallyHiddenComponent,
     } = this.props;
 
-    if (MenuUtil.includes(value, option.props.value)) {
-      onDeselect(option.props.value, option);
+
+    const selectedTxt = intl.formatMessage({ id: 'Terra.form.select.selected' });
+    const unselectedTxt = intl.formatMessage({ id: 'Terra.form.select.unselected' });
+    const shouldUnselectOption = MenuUtil.includes(value, option.props.value);
+    const optionATClickText = shouldUnselectOption ? unselectedTxt : selectedTxt;
+
+    if (visuallyHiddenComponent && visuallyHiddenComponent.current) {
+      visuallyHiddenComponent.current.innerHTML = `${option.props.display} ${optionATClickText}`;
```

Comment:
This is probably a question for the translations team or someone more familiar with translations, but the grammar in different languages may not always be in this order. 

`Selected` for example has a parameterized version: https://github.com/cerner/terra-core/blob/master/packages/terra-form-select/translations/en-US.json#L8

Reply:
I agree. When I copied this from https://github.com/cerner/terra-core/blob/master/packages/terra-form-select/src/multiple/Menu.jsx#L375, I thought it was odd that we weren't translating like:

```js
const unselectedText = intl.formatMessage({ id: 'Terra.form.select.unselected',  { text: option.props.display } });
```

Reply:
Translations added and fix made